src/ensembl/src/shared/components/loader/SidebarLoader.scss

```diff
@@ -0,0 +1,43 @@
+@import 'src/styles/common';
+
+.sidebarLoader {
+  box-sizing: border-box;
+  display: flex;
+  margin: 12px 0 0 10px;
+  flex-direction: column;
+  height: 100%;
+
+  section {
+    margin-top: 45px;
+  }
+
+  div {
+    background-color: $light-grey;
+    border-radius: 20px;
+    height: 9px;
+    margin-bottom: 10px;
+    animation: fade-in 0.5s ease-in;
```

Comment:
Hi Imran, fade-in just happen once. Wonder if we should be repeating it to convey that something is happening?

Reply:
It's supposed to happen only once and then stay put